blueocean-dashboard/src/main/resources/jenkins/plugins/blueocean/dashboard/Messages_zh.properties

```diff
@@ -0,0 +1,305 @@
+#创建credential
+creation.git.create_credential.button_create=\u521b\u5efa\u0063\u0072\u0065\u0064\u0065\u006e\u0074\u0069\u0061\u006c
+#创建credential发生错误. 请再尝试一次.
+creation.git.create_credential.error_msg=\u521b\u5efa\u0063\u0072\u0065\u0064\u0065\u006e\u0074\u0069\u0061\u006c\u53d1\u751f\u9519\u8bef\u002e\u0020\u8bf7\u518d\u5c1d\u8bd5\u4e00\u6b21\u002e
+#Credential类型
+creation.git.create_credential.credential_type=\u0043\u0072\u0065\u0064\u0065\u006e\u0074\u0069\u0061\u006c\u7c7b\u578b
+creation.git.create_credential.credential_type_ssh_key=SSH Key
+#用户名 & 密码
+creation.git.create_credential.credential_type_user_pass=\u7528\u6237\u540d\u0020\u0026\u0020\u5bc6\u7801
+#请输入一个有效的密码
+creation.git.create_credential.password_error=\u8bf7\u8f93\u5165\u4e00\u4e2a\u6709\u6548\u7684\u5bc6\u7801
+#密码
+creation.git.create_credential.password_title=\u5bc6\u7801
+#请输入一个有效的 SSH private key.
+creation.git.create_credential.sshkey_error=\u8bf7\u8f93\u5165\u4e00\u4e2a\u6709\u6548\u7684\u0020\u0053\u0053\u0048\u0020\u0070\u0072\u0069\u0076\u0061\u0074\u0065\u0020\u006b\u0065\u0079\u002e
+creation.git.create_credential.sshkey_title=SSH Private Key
+#请输入一个有效的用户名
+creation.git.create_credential.username_error=\u8bf7\u8f93\u5165\u4e00\u4e2a\u6709\u6548\u7684\u7528\u6237\u540d
+#用户名
+creation.git.create_credential.username_title=\u7528\u6237\u540d
+#添加credential
+creation.git.create_credential.title=\u6dfb\u52a0\u0063\u0072\u0065\u0064\u0065\u006e\u0074\u0069\u0061\u006c
+# creation for git
+#创建Pipeline
+creation.git.step1.create_button=\u521b\u5efa\u0050\u0069\u0070\u0065\u006c\u0069\u006e\u0065
+#正在创建Pipeline...
+creation.git.step1.create_button_progress=\u6b63\u5728\u521b\u5efa\u0050\u0069\u0070\u0065\u006c\u0069\u006e\u0065\u002e\u002e\u002e
+#添加
+creation.git.step1.create_credential_button=\u6dfb\u52a0
+creation.git.step1.credentials=Credentials
+#请选择一个有效的credential.
+creation.git.step1.credentials_error_invalid=\u8bf7\u9009\u62e9\u4e00\u4e2a\u6709\u6548\u7684\u0063\u0072\u0065\u0064\u0065\u006e\u0074\u0069\u0061\u006c\u002e
+creation.git.step1.credentials_placeholder=System Default
+#任何包含Jenkinsfile的仓库将会自动构建. 你想干什么?
+creation.git.step1.instructions=\u4efb\u4f55\u5305\u542b\u004a\u0065\u006e\u006b\u0069\u006e\u0073\u0066\u0069\u006c\u0065\u7684\u4ed3\u5e93\u5c06\u4f1a\u81ea\u52a8\u6784\u5efa\u002e\u0020\u4f60\u60f3\u5e72\u4ec0\u4e48\u003f
+#更多Jenkinsfile's介绍.
+creation.git.step1.instructions_link=\u66f4\u591a\u004a\u0065\u006e\u006b\u0069\u006e\u0073\u0066\u0069\u006c\u0065\u0026\u0023\u0033\u0039\u003b\u0073\u4ecb\u7ecd\u002e
+#请输入一个有效的URL
+creation.git.step1.repo_error_invalid=\u8bf7\u8f93\u5165\u4e00\u4e2a\u6709\u6548\u7684\u0055\u0052\u004c
+#请输入一个URL.
+creation.git.step1.repo_error_required=\u8bf7\u8f93\u5165\u4e00\u4e2a\u0055\u0052\u004c\u002e
+#仓库URL
+creation.git.step1.repo_title=\u4ed3\u5e93\u0055\u0052\u004c
+#连接到Git仓库
+creation.git.step1.title=\u8fde\u63a5\u5230\u0047\u0069\u0074\u4ed3\u5e93
+#保存
+creation.git.step2.button_save=\u4fdd\u5b58
+#'{0}'可用
```

Comment:
Original string is "Success! '{0}' is available.", but the translation doesn't include "Success!"